### ISSUE TITLE
Cannot focus on 2nd and 3rd linked content blocks with slick-slide classes.

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/extension-workshop#155]
The site should be keyboard accessible, it's not currently possible to focus beyond the first content link that has in `slick-slide` class-names.

![Extension_Workshop___Firefox_Extension_Workshop__Make_the_web_do_what_you_want__If_you_can_dream_it__our_open_souce_API_and_your_HTML__CSS__and_JavaScript_skills_can_make_it_happen_](https://user-images.githubusercontent.com/1514/55546153-95e77c80-56c6-11e9-8ece-24951b5ebbcc.jpg)

## STR:

Try tabbing into the content after the "Cool Things Extensions Can Do" and the content after the "Latest Developer News" heading.

## What happens: 

You can only tab into the first block, the 2nd and 3rd blocks are not focusable.

## What should happen:

All blocks should be focusable as they are links.

If this behaviour is happening because of a carousel component, since we're not currently using that maybe it can be removed?

**Comment by ioanarusiczki** [comment]
The same for what's after Latest Developer News - FF66(Win10)

![keyboard navigation](https://user-images.githubusercontent.com/33448286/56492582-e9dec780-64f4-11e9-9ba6-ad4b61a711ba.png)

**Comment by muffinresearch** [comment]
This is still an issue for the new design.

**Comment by ioanarusiczki** [comment]
@muffinresearch Yes, I can confirm that with FF68 (Win10).

**Comment by muffinresearch** [comment]
This appears to be a known issue with the dependency, they have open patches with fixes that may help with this:

* https://github.com/kenwheeler/slick/pull/3773
* https://github.com/kenwheeler/slick/pull/3716
